package.json

```diff
@@ -28,23 +28,29 @@
       "url": "https://github.com/nikoskalogridis"
     }
   ],
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
+  "main": "./v3/index.js",
+  "module": "./v3/index.mjs",
+  "types": "./v3/index.d.ts",
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
-      "require": "./dist/index.js",
-      "import": "./dist/index.mjs"
+      "types": "./v3/index.d.ts",
+      "require": "./v3/index.js",
+      "import": "./v3/index.mjs"
+    },
+    "./v3": {
+      "types": "./v3/index.d.ts",
+      "require": "./v3/index.js",
+      "import": "./v3/index.mjs"
     },
     "./v4": {
-      "types": "./dist/v4/index.d.ts",
-      "require": "./dist/v4/index.js",
-      "import": "./dist/v4/index.mjs"
+      "types": "./v4/index.d.ts",
+      "require": "./v4/index.js",
+      "import": "./v4/index.mjs"
```

Comment:
dev: continue declaring v4 exports for newer node versions and module resolutions.